templates/agenda/index.html.twig

```diff
@@ -0,0 +1,120 @@
+                    {{ form_widget(filter_form.status) }}
+                </div>
+                <div class="col-auto ml-2">
+                    {# Todo: Just reset filter rather than refresh page although it yields the same result #}
+                    <a name="" id="" class="btn btn-secondary" href="{{ path('agenda_index') }}" role="button">{% trans %}Reset{% endtrans %}</a>
+                </div>
+            </div>
+            {{ form_rest(filter_form) }}
+        </form>
+
+        <div id="js-process-spinner" class="spinner-border spinner-border-sm" style="display: none" role="status">
+            <span class="sr-only">Loading</span>
+        </div>
+
+        <div class="table-responsive-sm mt-3 js-process-table-results">
+            <table class="table table-striped table-hover">
+                <thead>
+                <tr>
+{#                    <th scope="col">{% trans %}Board{% endtrans %}</th>#}
+                    <th{% if pagination.isSorted('a.Board') %} class="sorted"{% endif %}>
+                        {{ knp_pagination_sortable(pagination, 'Board'|trans({}, 'agenda'), 'board.name') }}
+                    </th>
+                    <th{% if pagination.isSorted('a.Date') %} class="sorted"{% endif %}>
+                        {{ knp_pagination_sortable(pagination, 'Date'|trans({}, 'agenda'), 'a.date') }}
+                    </th>
+                    <th scope="col">{% trans %}From{% endtrans %}</th>
+                    <th scope="col">{% trans %}To{% endtrans %}</th>
+                    <th{% if pagination.isSorted('a.Status') %} class="sorted"{% endif %}>
+                        {{ knp_pagination_sortable(pagination, 'Status'|trans({}, 'agenda'), 'a.status') }}
+                    </th>
+                    <th class="text-right" scope="col" data-orderable="false">{% trans %}Function{% endtrans %}</th>
+                </tr>
+                </thead>
+                <tbody>
+                {% for agenda in pagination %}
+                    <tr>
+                        <td>{{ agenda.board|e }}</td>
+                        <td>{{ agenda.date|date('j F Y') }}</td>
+                        <td>{{ agenda.start|date('H:i') }}</td>
+                        <td>{{ agenda.end|date('H:i') }}</td>
+                        <td>{{ agenda.status|trans({}, 'agenda_status') }}</td>
+                        <td class="text-right">
+                            <div class="btn-group" role="group" aria-label="{% trans %}List functions{% endtrans %}">
+                                <!-- Show button -->
+                                <a href="{{ url('agenda_show', {id: agenda.id}) }}" role="button" class="btn btn-primary" data-toggle="tooltip" data-placement="auto" title="{% trans %}Show agenda{% endtrans %}"><i class="fas fa-eye"></i><span class="sr-only">{% trans %}Show agenda{% endtrans %}</span></a>
+                                <!-- Delete button -->
+                                {% if 3 == agenda.status %}
```

Comment:
`3` should probably be removed by a constant ore a method on the agenda class.

Reply:
Added and used `isReady` function to `Agenda` class.